sky/utils/cli_utils/status_utils.py

```diff
@@ -54,6 +54,7 @@ def show_status_table(cluster_records: List[Dict[str, Any]],
         StatusColumn('HOURLY_PRICE', _get_price, show_by_default=False),
         StatusColumn('STATUS', _get_status),
         StatusColumn('AUTOSTOP', _get_autostop),
+        StatusColumn('TOTAL_COST', _get_total_cost),
```

Comment:
nit: change the name to `COST (est.)` to claim that the cost is estimated and can be different from the actual cost?